src/analysis/retail/warlock/destruction/modules/features/Backdraft.tsx

```diff
@@ -36,8 +37,12 @@ class Backdraft extends Analyzer {
 
   constructor(options: Options) {
     super(options);
-    this._maxStacks = this.selectedCombatant.hasTalent(SPELLS.FLASHOVER_TALENT.id) ? 4 : 2;
-    this._stacksPerApplication = this.selectedCombatant.hasTalent(SPELLS.FLASHOVER_TALENT.id)
+    this._maxStacks = this.selectedCombatant.hasTalent(TALENTS.IMPROVED_CONFLAGRATE_TALENT.id)
+      ? 4
+      : 2;
+    this._stacksPerApplication = this.selectedCombatant.hasTalent(
+      TALENTS.IMPROVED_CONFLAGRATE_TALENT.id,
+    )
```

Comment:
doesn't necessarily need a fix right now bc this stage is more about making it compile than anything else, but this talent appears to no longer increase Backdraft stacks.

Reply:
Yep you're right. I tested in PTR to ensure that's the case